concourse/templates/default.mako

```diff
@@ -385,7 +385,7 @@ else:
 % elif job_step.name == 'update_component_dependencies':
         ${update_component_deps_step(job_step=job_step, job_variant=job_variant, github_cfg_name=github.name(), indent=8)}
 % elif job_step.name == 'publish':
-    ${publish_step(job_step=job_step, job_variant=job_variant)}
+        ${publish_step(job_step=job_step, job_variant=job_variant, indent=0)}
```

Comment:
I understand too less/nothing of it, but shouldn't this be
```suggestion
        ${publish_step(job_step=job_step, job_variant=job_variant, indent=8)}
```

Reply:
maybe.. however: before this change, this seems to have worked w/o using the indent function at all. so I wonder, why it should now suddenly be required to use it? Oo

Reply:
There are some strange things going on in this template. That 0-level indent effectively starts at level 6 (check the position of the `# noop` in my previous comment. It should be at level 0).

We really should refactor this template sometime. I _think_ it's in our backlog somewhere.

Reply:
> We really should refactor this template sometime. I think it's in our backlog somewhere.

LOL :)